### ISSUE TITLE
Clear form not working as expected on Existing Objects

### ISSUE THREAD

**Issue by DiegoPino** [esmero/webform_strawberryfield#45]
# What is this?

What the title of the issue says. Explicit

**Comment by DiegoPino** [comment]
@alliomeria can you test this on Monday? Wonder if this is still true and what can be do? I feel "clearing a form" on an existing Object is a bad idea anyway.

**Comment by alliomeria** [comment]
Just tested for an object on studio instance. Still seems to be an issue, Clear Form still not working as expected--values not cleared, and for certain steps on the default webform, Clear From button redirects you to the first step. Also agree that clearing an entire form's worth of data for existing objects seems like a bad idea, but perhaps there is a use case out there where this could be helpful.

**Comment by DiegoPino** [comment]
Ok. We need to deal with this. Thanks. Will look at the code and see what i can do. Appreciate the testing!

**Comment by DiegoPino** [comment]
Also: I feel clearing needs to go back to the first step (not a bug, a feature) because if we empty all data we can not ensure empty previous steps are Ok.

**Comment by alliomeria** [comment]
Agreed, clearing should go back to the first step. Though I can also see people expecting to just clear the values for a particular step on a form--say the subjects/linked data. Maybe an "are you sure you want to clear all the data for this object" popup wouldn't be amiss?

**Comment by DiegoPino** [comment]
Good idea. A confirmation step. I think I can do that via JS. Thanks
